CHANGELOG.md

```diff
@@ -0,0 +1,59 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [v1.0.0](https://github.com/betadots/pdc/tree/v1.0.0) (2023-11-28)
```

Comment:
We move from v8 to v1?

Reply:
same naming pattern and build as in puppetserver, the older tags were, complicated